homeassistant/components/cloud/http_api.py

```diff
@@ -206,19 +222,57 @@ async def post(self, request: web.Request) -> web.Response:
 class CloudLoginView(HomeAssistantView):
     """Login to Home Assistant cloud."""
 
+    _mfa_tokens: dict[str, str] = {}
```

Comment:
Side note: Since this class will only be instantiated once I think this ok. Generally, initializing mutable objects as class attributes is error prone.